File: JXCategoryView/Example/PagingView/Example/HobbyListView.m
//
//  HobbyListView.m
//  JXCategoryView
//
//  Created by jiaxin on 2018/8/27.
//  Copyright © 2018年 jiaxin. All rights reserved.
//

#import "HobbyListView.h"

@implementation HobbyListView

- (instancetype)initWithFrame:(CGRect)frame
{
    self = [super initWithFrame:frame];
    if (self) {
        self.dataSource = @[@"吃烤肉", @"吃鸡腿肉", @"吃牛肉", @"各种肉"];
    }
    return self;
}

@end

File: JXCategoryView/Example/PagingView/Example/HobbyListView.swift
//
//  HobbyListView.swift
//  JXPagingView
//
//  Created by jiaxin on 2018/5/29.
//  Copyright © 2018年 jiaxin. All rights reserved.
//

import UIKit

class HobbyListView: TestListBaseView {

    override init(frame: CGRect) {
        super.init(frame: frame)

        dataSource = ["吃烤肉", "吃鸡腿肉", "吃牛肉", "各种肉"]
    }

    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }

}
